public/firmwares/manifest.json

```diff
@@ -25,12 +35,6 @@
       "type": "ot-rcp",
       "version": "2.4.4.0"
     },
-    {
-      "name": "Multiprotocol (experimental)",
```

Comment:
We're removing firmware flashing from the multiprotocol addon so I think it would be good to keep around multiprotocol firmware installation in the web flasher. What about renaming it to `Multiprotocol (deprecated)`?